.github/workflows/setup/action.yml

```diff
@@ -0,0 +1,46 @@
+name: _Setup
+on:
+  workflow_call:
+    registry-url:
+      required: false
+      type: string
+
+env:
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+
+jobs:
+  setup:
+    name: Checkout and setup
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        id: checkout
+        uses: actions/checkout@v3
+      - name: Setup Node
+        id: setup-node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: '.npm'
+          registry-url: ${{ inputs.registry-url }}
+      - name: Node version
```

Comment:
Debug statements?

Reply:
what do you mean?

Reply:
Haha, you're right. Those are probably my debug statements :)